src/core/generateZodSchema.ts

```diff
@@ -447,6 +455,25 @@ function buildZodPrimitive({
     }
   }
 
+  // Deal with enums used as literals
+  if (
```

Comment:
Let's add a small test in `generateZodSchema.test.ts`

Reply:
https://github.com/fabien0102/ts-to-zod/pull/40#discussion_r678971858